build-ci/platforms.rst

```diff
@@ -0,0 +1,46 @@
+###################
+Supported Platforms
+###################
+
+Introduction
+============
+
+The LSST Data Management software is required to be portable across many compute platforms ranging from high-performance computing centres to laptops (DMS-REQ-0308).
+The software will likely work on any Unix-like operating system but we limit our testing and support to a fixed set of operating systems.
+
+Patches may be accepted from developers who test on other operating systems.
+
+Definitions
+===========
+
+We use the following definitions when defining which operating systems are supported.
+
+* A *supported* operating system is one which is regularly tested by our continuous integration system and which we will ensure all tests pass.
+  The list of supported operating systems may change over time, an in particular, if we encounter difficulties where keeping an operating system running is taking excessive developer and CI resources, we reserve the right to drop it from the list.
+  Each supported operating system will be associated with a specific compiler that we use for testing.
```

Comment:
We should be testing using multiple compilers on CentOS, including clang and icc.

Reply:
yes, but we aren't at the moment. Maybe if we dropped centos6 we could add a clang job instead.